### DISCUSSION TITLE
Could I launch Kopf in a framework

### DISCUSSION THREAD

**Discussion by lperdereau** [nolar/kopf#745] [Q&A] [answered]
Hello,

I'm trying to use Kopf with [Django](https://www.djangoproject.com) but in your documentation I only see you use `Kopf run` is there another way to do it ?

Best regards,

**Comment by nolar** [comment, marked as answer]
Kopf is embeddable into other applications as well: https://kopf.readthedocs.io/en/stable/embedding/

**Reply by lperdereau** [reply]
Thank you I didn't see it :)
